grid/example/src/cells/RowOptions.js

```diff
@@ -18,7 +20,17 @@ const RowOptions = memo((props) => {
         setOpen(false);
     };
 
-    return (
+    const showEditOverLay = () => {
+        console.log("showEditOverLay");
+        setOpen(false);
+        setOverLayOpen(true)
+    }
+
+    const setOverLayClose = () => {
+        setOverLayOpen(false)
+    }
+
+    let htmlReturnValue = isOverLayOpen === false ?

```

Comment:
Let's use const instead of let, if that value is not getting changed anywhere

Reply:
used const